java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -133,7 +133,7 @@ class ArcHostContextParticle(
         particleName: String,
         location: String
     ): Particle = instantiatedParticles.getOrPut(particleName) {
-        instantiateParticle(ParticleIdentifier.from(location))
+        instantiateParticle(ParticleIdentifier.from(location), null)
```

Comment:
You should put some comment here on why it's null and a reference to the bug :)

Reply:
Thanks!